book/src/prerequisites.md

```diff
@@ -1,21 +1,73 @@
-# Installation
+# Prerequisites
```

Comment:
Should this include a paragraph about getting GTK4 and installing the Rust toolchain (e.g. via rustup)?

Reply:
I mean, separate from the `cargo` / `cargo + meson` sections

Reply:
I thought about that. 
However, they are only two sentences and I am afraid that the overview gets worse if I put this separate.